musicbot/config.py

```diff
@@ -42,6 +42,7 @@ def __init__(self, config_file):
 
         self.owner_id = config.get('Permissions', 'OwnerID', fallback=ConfigDefaults.owner_id)
         self.dev_ids = config.get('Permissions', 'DevIDs', fallback=ConfigDefaults.dev_ids)
+        self.bot_exception_ids = config.get("Permissions", "BotExceptionIDs", fallback=ConfigDefaults.bot_exception_ids)
```

Comment:
You need to process ids after this as this will get str, not set of ints. Preferably process it in run_checks